pkg/containerd/Dockerfile

```diff
@@ -12,3 +12,4 @@ COPY --from=containerd-dev /usr/bin/containerd /usr/bin/ctr /usr/bin/containerd-
 COPY --from=alpine /usr/share/zoneinfo/UTC /etc/localtime
 COPY --from=alpine /etc/init.d/ /etc/init.d/
 COPY etc etc/
+COPY lib/apk lib/apk/
```

Comment:
I think you meant this to be `COPY --from=build /lib/apk /lib/apk`

Same for the others

Reply:
@deitch updated PR with --from